acceptance/openstack/messaging/v2/message_test.go

```diff
@@ -0,0 +1,21 @@
+package v2
```

Comment:
One thing I totally missed earlier is that a build tag and blank line need to be added at the top of acceptance tests:

Something like:

```go
// +build acceptance messaging messages

package v2
```

If you could update the `queue_test.go` file, it would save me a PR.

The `messaging.go` file does not need this.

The build tag is to exclude the acceptance tests from running during a normal `go test`. The script that Travis runs automatically excludes acceptance tests, but any other person who runs `go test ./...` would trigger the acceptance tests.

I apologize about not catching that before.